src/vector/v.scatterplot/v.scatterplot.html

```diff
@@ -127,6 +153,47 @@ <h3>Example 3</h3>
 pca_1 against pca_1. Colors represent the land use categories in the 
 point locations based on the landclass96 map.</em>
 
+<h3>Example 4</h3>
+
+<p>
+Rename the PCA layers to remove the dots from the name. Next, use the 
+<a 
+href="https://grass.osgeo.org/grass82/manuals/addons/v.what.rast.label.html">v.what.rast.label</a> 
+addon to sample the values of the raster layers <i>pca.1</i> and 
+<i>pca.2</i>, and the values + labels of the <i>landclass96</i>. Add a 
+column with the landclass colors using <i>v.colors</i>.
+
+<div class="code"><pre>
+g.rename raster=pca.1,pca_1
+g.rename raster=pca.2,pca_2
+v.what.rast.label vector=randompoints raster=landclass96 \
+         raster2=pca_1,pca_2 output=randompoints2
+v.colors map=randompoints2 use=attr column=landclass96_ID \
+         raster=landclass96 rgb_column=RGB
+</pre>
+</div>
+
+<p>
+Extract the points with the categories forest (5), water (6) and 
+developed (1). Create a scatterplot of pca_1 against pca_2 and add the 
+2 SD confidence ellipse of the covariance of the two variables for 
+each of the land use categories, coloring both the dots and ellipses 
+using the landclass colors.
+
+<div class="code"><pre>
+v.extract input=randompoints2 \
+          where='landclass96_ID=1 OR landclass96_ID=5 OR landclass96_ID=6' \
+          output=forwatdev
+v.scatterplot -e map=forwatdev x=pca_1 y=pca_2 rgbcolumn=RGB s=5  \
+              groups=landclass96 groups_rgb=RGB 
+</pre>
+</div>
+
+<p>
+<img src="v_scatterplot_04.png"><br> <em>Figure 4. Scatterplot with 
+confidence ellipses per land class. The radius of the ellipses is 2 SD.</em>
```

Comment:
```suggestion
<div align="center" style="margin: 10px">
<a href="v_scatterplot_04.png">
<img src="v_scatterplot_04.png" alt="v.scatterplot: confidence ellipses per land class" border="0">
</a><br>
<i>Figure 4. Scatterplot with confidence ellipses per land class.
 The radius of the ellipses is 2 SD.</i>
</div>
```

Compare https://trac.osgeo.org/grass/wiki/Submitting/Docs#Images